src/components/App.tsx

```diff
@@ -2109,15 +2109,6 @@ class App extends React.Component<ExcalidrawProps, AppState> {
 
     this.updateGestureOnPointerDown(event);
 
-    // fixes pointermove causing selection of UI texts #32
-    event.preventDefault();
-    // Preventing the event above disables default behavior
-    // of defocusing potentially focused element, which is what we
-    // want when clicking inside the canvas.
-    if (document.activeElement instanceof HTMLElement) {
-      document.activeElement.blur();
-    }
-
```

Comment:
Just need to make sure nothing breaks due to this removal else pr looks good

Reply:
Definitely do check. I wouldn't be surprised if something did :).